portfolio/src/main/webapp/recent.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
```

Comment:
The indentation in this file seems inconsistent and not according to the style guide https://google.github.io/styleguide/htmlcssguide.html#HTML_Formatting_Rules.

Basically, indent 2 spaces at a time, indent child elements, and make sure that <div ..> are at the same indentation level as the closing </div>

There also shouldn't be whitespace next to the = signs.

If you use a linter in your IDE, this should catch a lot of these style issues.

I encourage to use a linebreak between headers/paragraphs as well since it will make it easier to read.

Reply:
Fixed in 7f183ec.